ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -71,29 +71,29 @@ let getHeadmarkerId = (data, matches) => {
             return {
               en: 'Tank Busters',
               de: 'Tank buster',
-              fr: 'Tank busters',
+              fr: 'Tank buster',
```

Comment:
We had a discussion on a post on this subject. It is not fundamentally incorrect the plural (s) in French. but tank buster is an import word in French which does not really have a faithful translation. In addition, it is almost common for either a boss to put several buster to sharpen or that 2 bosses strike with a buster one after the other. So in French, we left on tank buster in all situations, because the players know the mechanics. I would add that in some files, the plural is not always present or that there is just buster or busters. So we started on this writing for all, singular or plural. Sorry for the long paragraph.

Reply:
> I would add that in some files, the plural is not always present or that there is just buster or busters. 

This should be fixed.  It should be plural buster when there are multiple busters, in my opinion.